01week/resume/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html> <head>
+        <meta charset="UTF-8">
+        <meta name="description" content="Free Web tutorials">
+        <meta name="keywords" content="HTML,CSS,XML,JavaScript">
+        <meta name="author" content="John Doe">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" type="text/css" href="./css/style.css">
+    </head>
+<body>
+    
+<title>Elvira Joubert - Resume</title>
+<h1>Elvira Joubert - Resume</h1>
+
+<h4> Austin, Texas &#9830 512 698 0964 &#9830 
+    <a href="mailto:reagent01@gmail.com?Subject=Hello%20again"target="_top">Send Mail</a> 
```

Comment:
by looking at this. i can see the different color text which means there is probably something missing in your code. closing tag or something. This is one way the text editor trying to tell you something is wrong/missing